src/bridge/index.ts

```diff
@@ -135,4 +141,34 @@ export class BridgeService {
       ),
     };
   }
+
+  private async prepareApproveParams(
```

Comment:
If you use int amount, this method wil be simpler

Reply:
I changed `approve` method to accept an integer amount